styles/pages/impact.module.scss

```diff
@@ -0,0 +1,119 @@
+@import '../variables.scss';
+
+.container {
+  margin: 0 auto;
+  max-width: 50rem;
+  padding: 0 1em;
+}
+
+.pronouns {
+  margin-top: 0;
+  padding-top: 0;
+}
+
+.name {
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+
+.text-center-holographic {
+  z-index: -100;
+  position: absolute;
+  left: 626.37px;
+  top: 605.3px;
+  padding-left: 3px;
+  text-align: center;
+  font-family: $font-header;
+  background-image: linear-gradient(rgba(255, 0, 92, 0.3996),
+  rgba(177, 100, 255, 0.54),
+  rgba(51, 255, 206, 0.54),
+  rgba(125, 255, 46, 0.54));
+  background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.workshop-card {
+  padding: 0 1em;
+}
+
+.workshop-image-container {
+  // TODO: let's deal with this later :)
+  // border: 1px solid gray;
+  // border-radius: 10px;
+  // filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  max-width: 300px;
+}
+
+.blog-card {
+  background-blend-mode: multiply;
+  background-color: rgba(0, 0, 0, 0.25);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 10px;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  color: white;
+  display: flex;
+  min-height: 300px;
+  padding: 2em;
+  text-decoration: none;
+}
+
+.blog-card-content {
+  align-self: flex-end;
+}
+
+.blog-title {
+  font-family: $font-header;
+  font-size: 1.5rem;
+  margin: 0;
+}
+
+.blog-info {
+  display: flex;
+  font-size: 1rem;
+}
+
+.blog-author-img {
+  border-radius: 9999px;
+  display: inline-block;
+  height: 3em;
+  margin-right: 1em;
+  width: 3em;
+}
+
+.carousel-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.button-container {
+  margin: 5px;
+  padding: 5px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.btn-selected {
+  padding: 5px;
+  margin: 5px;
+  display: inline-block;
+  background-color: #49EEA1;
+  border-radius: 50%;
+  width: 5.73px;
+  height: 5.73px;
+  border: none;
+}
+
+.btn-unselected {
+  padding: 5px;
+  margin: 5px;
+  display: inline-block;
+  background-color: #C4C4C4;
+  border-radius: 50%;
+  width: 5.73px;
+  height: 5.73px;
+  border: none;
+}
```

Comment:
Two things:

1. I think we can simplify a bit of this CSS by having a base class, and then just toggling on the things we need to change (in this case, *just* the `background-color`) - that way, we can duplicate less code!
2. Can we add some styling here to make the cursor hover change to typical button behaviour? ex `cursor:pointer` with the `:hover` selector?